docs/reference/search/field-caps.asciidoc

```diff
@@ -34,6 +34,10 @@ GET /_field_caps?fields=rating
 The field capabilities API returns the information about the capabilities of
 fields among multiple indices.
 
+Use the field capabilities API to return <<runtime,runtime fields>> like any
```

Comment:
More than "use the API to return runtime fields", I would say "the fiels caps API returns runtime fields like any ordinary field

Reply:
I'll shorten this statement to:
>The field capabilities API returns runtime fields like any other field.